Sources/ProjectDescription/TestAction.swift

```diff
@@ -112,33 +88,28 @@ public struct TestAction: Equatable, Codable {
     ///   - testPlans: List of test plans to run.
     ///   - arguments: Arguments passed when running the tests.
     ///   - configuration: Configuration to be used.
-    ///   - coverage: Whether test coverage should be collected.
-    ///   - codeCoverageTargets: The targets the test coverage should be collected from.
     ///   - expandVariableFromTarget: A target that will be used to expand the variables defined inside Environment Variables definition
     ///   - preActions: Actions to execute before running the tests.
     ///   - postActions: Actions to execute after running the tests.
+    ///   - options: Test options.
     ///   - diagnosticsOptions: Diagnostics options.
-    ///   - language: The language to be used.
-    ///   - region: The region to be used.
     /// - Returns: An initialized test action.
     public static func testPlans(_ testPlans: [Path],
                                  configuration: ConfigurationName = .debug,
                                  preActions: [ExecutionAction] = [],
+                                 options: TestActionOptions = .options(),
```

Comment:
For the test plans case I don't believe we can specify any options :/

![Screenshot 2021-09-16 at 14 59 15](https://user-images.githubusercontent.com/11914919/133625855-7154c750-9df1-4bf7-bb50-133c771a4924.png)

We can make it optional to support both the `targets` and `testPlans` cases